app/src/main/java/org/mozilla/focus/settings/permissions/SitePermissionsFragment.kt

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package org.mozilla.focus.settings.permissions
+
+import android.content.SharedPreferences
+import android.os.Bundle
+import androidx.preference.Preference
+import org.mozilla.focus.R
+import org.mozilla.focus.ext.requireComponents
+import org.mozilla.focus.ext.requirePreference
+import org.mozilla.focus.settings.BaseSettingsFragment
+import org.mozilla.focus.state.AppAction
+import org.mozilla.focus.state.Screen
+
+class SitePermissionsFragment :
+    BaseSettingsFragment(),
```

Comment:
Site permissions screen was added to follow the Fenix design since from [this discussion](https://github.com/mozilla-mobile/focus-android/discussions/5830) it seems that all permissions are going to be added

Reply:
Thanks for the clarification!